templates/default/start-nm.sh.erb

```diff
@@ -5,6 +5,20 @@ h=$(hostname)
 bin=$(dirname "${BASH_SOURCE-$0}")
 bin=$(cd "$bin"; pwd)
 
+if [ <%= node['install']['localhost'] %> == "true" ] ; then
+    systemctl status nodemanager
```

Comment:
`systemctl status` might not return and needs user intervention. In my VM that't the behaviour. Use `systemctl is-active` instead